src/chunktransformer.ts

```diff
@@ -15,10 +15,44 @@
  */
 import {Transform, TransformOptions} from 'stream';
 
+import {Family} from './family';
```

Comment:
I don't think we want to pull this in, looks like family should be something like

```ts
interface Family {
  [qualifier: string]: Qualifier[];
}
```

Reply:
Perfect, had doubts myself, thanks Dave!